{{cookiecutter.__src_folder_name}}/.devcontainer/docker-compose_dev.yml

```diff
@@ -1,28 +1,38 @@
-# This overwrites select settings from the main docker-compose.yml.
-# WARNING: THIS SHOULD NOT BE USED DIRECTLY
+{# This section is for importing items #}
+{% if "postgres" in cookiecutter.db_resource %}
+  {% set db_app = "docker_compose_postgres.yml" %}
+{% endif %}
+{% if "mongodb" in cookiecutter.db_resource %}
+  {% set db_app = "docker_compose_mongodb.yml" %}
+{% endif %}
+{% from db_app import db, app_environment with context %}
+{# end import section #}
 version: '3'
 services:
-  # Update this to the name of the service you want to work with in your docker-compose.yml file
   db:
-    environment:
-      POSTGRES_USER: postgres
-      POSTGRES_DB: relecloud
-      POSTGRES_HOST: db
-      POSTGRES_PASSWORD: postgres
-      POSTGRES_PORT: 5432  
-
+    {{ db() }}
+  {# TODO: Create a way to add services such as redis, rabbitmq, etc. #}
```

Comment:
I think TODOs like this one (enhancements) are better as issues in the repo but no biggie.

Reply:
definitely and can add.